dune-project

```diff
@@ -36,9 +36,11 @@
   logs
   mtime
   yojson
+  tyxml
```

Comment:
This might be controversial. The addition of Tyxml as a dependency allows Opium to provide several helper functions (`Response.of_html`, `Response.of_svg`). I figured these are common enough that it justifies the addition of a new dependency.

Reply:
Not controversial at all. Tyxml is pretty popular and the install counts on opam justify adding helpers for tyxml in my opinion :)